src/Microsoft.OData.Core/JsonLight/ODataJsonLightWriter.cs

```diff
@@ -1597,7 +2636,6 @@ private void WriteDeltaLinkTarget(ODataDeltaLinkBase link)
         /// Validates that the ODataResourceSet.InstanceAnnotations collection is empty for the given expanded resource set.
         /// </summary>
         /// <param name="resourceSet">The expanded resource set in question.</param>
-        [SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic", Justification = "An instance field is used in a debug assert.")]

```

Comment:
Why did we stop suppressing this warning?

Reply:
@KenitoInc Because the `Debug.Assert` statements refer to `this` so I reasoned that the "MarkMembersAsStatic" warning would no longer valid. Unless the alternative reasoning is that we need to retain the suppression since assert statements are stripped in non-debug scenarios? What is your opinion?